puppet/zulip/manifests/memcached.pp

```diff
@@ -31,17 +31,29 @@
     content => zulipsecret('secrets', 'memcached_password', ''),
     notify  => Exec[generate_memcached_sasldb2],
   }
+  file { '/var/lib/zulip/memcached-sasldb2.stamp':
+    owner   => 'root',
+    group   => 'root',
+    mode    => '0644',
+    content => '1',
```

Comment:
I assume the idea here is we can update this to trigger a rebuild.  Makes sense.

Reply:
I suppose `/var/lib/zulip` is created by `puppet/zulip/manifests/nginx.pp` or `scripts/lib/setup-apt-repo`; that will work because `setup-apt-repo` is required, but we might want to factor out a `/var/lib/zulip` directory in `base.pp` since that's kinda fragile.